packages/cli/src/tools/processorType.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Returns if the processor is Intel or AMD
+ */
+export const getProcessorType = () => {
+  return process.env.PROCESSOR_IDENTIFIER!.includes('Intel') ? 'Intel' : 'AMD';
+};
```

Comment:
Perhaps move this function into a ‘windows’ namespace or, if you already foresee its usage on other platforms make it also actually check for `AMD` before returning that?

Reply:
I already have a `tools/windows` but this particular thing is not specific to the platform. Do you know if there are any particularities with Android, AMD, and Linux?

I don't expect to return other platforms (unless macOS starts running on ARM), but if you have access to an AMD processor I can explicitly check for the right value.

Reply:
I do not (on both counts). I guess I’m saying that, until it’s needed elsewhere _and tested there_, perhaps keep it in the namespace where we know it’s being used?

Reply:
Moved under `tools/windows`